porting/install-build/reference-rootfs.rst

```diff
@@ -2,10 +2,10 @@
 Halium reference rootfs
 =======================
 
-Once you have built the system.img from the android tree, you can download and install the rootfs using the ``halium-install`` script from the `halium-scripts repository <https://github.com/Halium/halium-scripts/>`_.
+Once you have built the system.img from the android tree, you can download and install the rootfs using the ``halium-install`` script from the `halium-scripts repository <https://github.com/Halium/halium-scripts/>`.
 
 
-Install hybris-boot.img
+Install hybris-boot.img 
```

Comment:
Please remove the space at the end of the line.

Reply:
Fixed.